docs/ovs-pipeline.md

```diff
@@ -377,34 +377,32 @@ In the future this table may support an additional mode of operations, in which
 it will implement kube-proxy functionality and take care of performing
 load-balancing / DNAT on traffic destined to services.
 
-### CnpEgressRuleTables (45-49)
+### AntreaPolicyEgressRuleTables (45, 49)
```

Comment:
I have noticed that the flows for the `DNAT` table are outdated:
```
2. table=40, priority=0 actions=goto_table:50
```

I believe it should read `goto_table:45`.

Would you mind updating it as part of this PR?

Reply:
Done. Thanks for spotting this.